src/js/tabs/pid_tuning.js

```diff
@@ -995,19 +995,19 @@ TABS.pid_tuning.initialize = function (callback) {
 
         if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
             FC.TUNING_SLIDERS.slider_pids_mode = parseInt($('#sliderPidsModeSelect').val());
-            FC.TUNING_SLIDERS.slider_master_multiplier = TuningSliders.sliderMasterMultiplier * 100;
-            FC.TUNING_SLIDERS.slider_roll_pitch_ratio = TuningSliders.sliderRollPitchRatio * 100;
-            FC.TUNING_SLIDERS.slider_i_gain = TuningSliders.sliderIGain * 100;
-            FC.TUNING_SLIDERS.slider_pd_ratio = TuningSliders.sliderPDRatio * 100;
-            FC.TUNING_SLIDERS.slider_pd_gain = TuningSliders.sliderPDGain * 100;
-            FC.TUNING_SLIDERS.slider_dmin_ratio = TuningSliders.sliderDMinRatio * 100;
-            FC.TUNING_SLIDERS.slider_ff_gain = TuningSliders.sliderFFGain * 100;
+            FC.TUNING_SLIDERS.slider_master_multiplier = Math.round((TuningSliders.sliderMasterMultiplier * 100)/5)*5;
+            FC.TUNING_SLIDERS.slider_roll_pitch_ratio = Math.round((TuningSliders.sliderRollPitchRatio * 100)/5)*5;
+            FC.TUNING_SLIDERS.slider_i_gain = Math.round((TuningSliders.sliderIGain * 100)/5)*5;
+            FC.TUNING_SLIDERS.slider_pd_ratio = Math.round((TuningSliders.sliderPDRatio * 100)/5)*5;
+            FC.TUNING_SLIDERS.slider_pd_gain = Math.round((TuningSliders.sliderPDGain * 100)/5)*5;
+            FC.TUNING_SLIDERS.slider_dmin_ratio = Math.round((TuningSliders.sliderDMinRatio * 100)/5)*5;
+            FC.TUNING_SLIDERS.slider_ff_gain = Math.round((TuningSliders.sliderFFGain * 100)/5)*5;
 
-            FC.TUNING_SLIDERS.slider_dterm_filter = TuningSliders.sliderDTermFilter ? 1 : 0;
-            FC.TUNING_SLIDERS.slider_dterm_filter_multiplier = TuningSliders.sliderDTermFilterMultiplier * 100;
+            FC.TUNING_SLIDERS.slider_dterm_filter = TuningSliders.sliderDTermFilter;
```

Comment:
Do we want to allow arbitrary values in here?

Reply:
rounded with x 20.